mt_sat/mt_sat_wrapper.m

```diff
@@ -221,6 +237,24 @@ function mt_sat_wrapper(mtw_nii,pdw_nii,t1w_nii,mtw_jsn,pdw_jsn,t1w_jsn,varargin
     savejson('',provenance,[pwd filesep 'MTsat.json']);
 end
 
+% JSON file for dataset_description
+addDescription = struct();
+addDescription.Name = 'qMRLab Outputs';
+addDescription.BIDSVersion = '1.5.0';
+addDescription.DatasetType = 'derivative';
+addDescription.GeneratedBy.Name = 'qMRLab';
+addDescription.GeneratedBy.Version = qMRLabVer;
```

Comment:
`qMRLabVer()` ?

Reply:
@agahkarakuzu in this case, the `addDescription.GeneratedBy.Version` field is the only one I could fill automatically because qMRLabVer is outputting the qMRLab version (e.g. [2.4.1]).

Reply:
@jvelazquez-reyes sorry, I was confused because `()` were missing from the function call. IIRC it returns an array, can we make sure that it is rendered nicely in the json output?

Reply:
@agahkarakuzu you're right, it returns an array. I checked and the json output is the same using `qMRLabVer` and `qMRLabVer()` but I will add `()` in the next commit.